QEfficient/transformers/models/mllama/modeling_mllama.py

```diff
@@ -72,6 +74,35 @@ def qeff_apply_rotary_pos_emb(q, k, cos, sin, position_ids, unsqueeze_dim=1):
     # Cast back to original dtype
     return q_embed.to(q.dtype), k_embed.to(k.dtype)
 
+
+def _prepare_cross_attention_mask(
```

Comment:
nit: this all functions can be imported from the mllama modeling directly to avoid code duplicates. pls check
or put in the common modeling_utils